src/objects/Term_Grade__c.object

```diff
@@ -41,7 +41,14 @@
         <type>Default</type>
     </actionOverrides>
     <allowInChatterGroups>false</allowInChatterGroups>
-    <compactLayoutAssignment>SYSTEM</compactLayoutAssignment>
+    <compactLayoutAssignment>Term_Grade_Compact_Layout</compactLayoutAssignment>
+    <compactLayouts>
```

Comment:
Lets remove the compactLayout section and leave it as SYSTEM. We've recently found out the compact layouts are push upgradeable but also editable by the subscriber, which means if they make changes, this will overwrite their changes on the next push upgrade. So let's leave the <compactLayoutAssignment>SYSTEM</compactLayoutAssignment> as is.